sdk/storage/azure-storage-file-share/src/test/resources/session-records/ModelTestsNTFSToAttributes2.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think the capitalization on the file name is causing playback failures in Linux, I think `NTFS` -> `Ntfs` is needed.